### PR TITLE
web/portal: Added Billable call to platform portal

### PR DESCRIPTION
**Pull request by fmendozzac**
#### Type Of Change 
- [ ] Small bug fix
- [x] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [x] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

#### Description
Added Billable call entity to platform portal

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

File: web/portal/platform/src/entities/BillableCall/BillableCall.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  },
+};
+
+const BillableCall: EntityInterface = {
+  ...defaultEntityBehavior,
+  icon: ChatBubbleIcon,
+  iden: 'BillableCall',
+  title: _('External call', { count: 2 }),
+  path: '/billable_calls',
+  toStr: (row: BillableCallPropertyList<EntityValue>) => row.id as string,
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/BillableCall/BillableCall.tsx:102]: id is a umber actually
```suggestion
  toStr: (row: BillableCallPropertyList<EntityValue>) => row.id.toString(),
```

File: web/portal/platform/src/entities/BillableCall/BillableCall.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+};
+
+const BillableCall: EntityInterface = {
+  ...defaultEntityBehavior,
+  icon: ChatBubbleIcon,
+  iden: 'BillableCall',
+  title: _('External call', { count: 2 }),
+  path: '/billable_calls',
+  toStr: (row: BillableCallPropertyList<EntityValue>) =>
+    row.id?.toString() as string,
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/BillableCall/BillableCall.tsx:103]:
```suggestion
    row.id?.toString(),
```

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: web/portal/platform/src/entities/ProxyUser/ProxyUser.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import SchemaIcon from '@mui/icons-material/Schema';
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/ProxyUser/ProxyUser.tsx:1]: We've two different PRs mixed here

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Pull request merged**